components/overflow-group/overflow-group-mixin.js

```diff
@@ -68,7 +66,7 @@ export const OverflowGroupMixin = superclass => class extends LocalizeCoreElemen
 	}
 
 	static get styles() {
-		return [offscreenStyles, css`
```

Comment:
I checked consumers of `OverflowGroupMixin` to make sure they weren't relying on this mixing including these.